resources/english.yml

```diff
@@ -1214,4 +1214,6 @@ msg_reclaim_available: '&aReclaim available.'
 
 #Added in 0.93
 msg_changed_peaceful: '&cTown neutrality/peacefullness is now %s.'
-msg_err_town_unclaim_canceled: '&cUnclaim canceled. You cannot unclaim this area at this time.'
\ No newline at end of file
+msg_err_town_unclaim_canceled: '&cUnclaim canceled. You cannot unclaim this area at this time.'
+
+msg_status_npc: '&c%s is not an real player, and is limited to Towny only.'
```

Comment:
If you want to be nit-picky on spelling, it should be "is not a real player" instead of "is not an real player"

Reply:
done